src/ngraph/op/fused/normalize.cpp

```diff
@@ -0,0 +1,146 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <algorithm>
+#include <iterator>
+
+#include "ngraph/builder/norm.hpp"
+#include "ngraph/op/divide.hpp"
+#include "ngraph/op/multiply.hpp"
+#include "ngraph/op/util/broadcasting.hpp"
+#include "ngraph/op/util/reshape.hpp"
+#include "normalize.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+op::Normalize::Normalize(const std::shared_ptr<ngraph::Node>& data,
+                         const std::shared_ptr<ngraph::Node>& scale,
+                         bool across_spatial,
+                         bool channel_shared,
+                         float eps)
+    : FusedOp("Normalize", {data, scale})
+    , m_across_spatial{across_spatial}
+    , m_channel_shared{channel_shared}
+    , m_eps{eps}
+{
+    constructor_validate_and_infer_types();
+}
+
+void op::Normalize::pre_validate_and_infer_types()
+{
+    const auto& data_pshape = get_input_partial_shape(0);
+    const auto& scale_pshape = get_input_partial_shape(1);
+
+    if (data_pshape.is_static() && scale_pshape.is_static())
+    {
+        const auto& data_shape{data_pshape.to_shape()};
+        const auto& scale_shape{scale_pshape.to_shape()};
+
+        // Input data must be 2, 3 or 4D tensor.
+        NODE_VALIDATION_CHECK(this,
+                              (data_shape.size() >= 2 && data_shape.size() <= 4),
+                              "Input tensor rank must be 2, 3 or 4 dimensional (actual input "
+                              "shape: ",
+                              data_shape,
+                              ").");
+        if (m_channel_shared)
+        {
+            NODE_VALIDATION_CHECK(this,
```

Comment:
Yes it can. But whether scale is shared across channels or each has separate scale is controlled by `channel_shared` attribute. There are already UTs for this.